internal/services/network/virtual_network_resource_test.go

```diff
@@ -500,11 +500,12 @@ resource "azurerm_resource_group" "test" {
 }
 
 resource "azurerm_virtual_network" "test" {
-  name                = "acctestvirtnet%d"
-  address_space       = ["10.0.0.0/16", "10.10.0.0/16"]
-  location            = azurerm_resource_group.test.location
-  resource_group_name = azurerm_resource_group.test.name
-  dns_servers         = ["10.7.7.2", "10.7.7.7", "10.7.7.1", ]
+  name                           = "acctestvirtnet%d"
+  address_space                  = ["10.0.0.0/16", "10.10.0.0/16"]
+  location                       = azurerm_resource_group.test.location
+  resource_group_name            = azurerm_resource_group.test.name
+  dns_servers                    = ["10.7.7.2", "10.7.7.7", "10.7.7.1", ]
+  private_endpoint_vnet_policies = "Basic"
```

Comment:
This has been set in the configuration for 3.x. Can you please set this in the configuration that will run in 4.x and also clean up the `complete` config by removing the 3.x config that isn't valid anymore?